desktop/components/my_active_bids/index.styl

```diff
@@ -37,6 +37,10 @@ item-margin-bottom = 16px
 .my-active-bids-item-details
   line-height 20px
 
+.my-active-bids-item-details--hide-bid
+  @extends .my-active-bids-item-details
+  padding-top 10px
+
```

Comment:
I imagine this will be rebuilt in React in due time and we can refactor then. This PR just addresses a few branching conditions that weren't present before.